docs/source/new-project.rst

```diff
@@ -11,26 +11,63 @@
 Quick Start
 ===========
 
-PyTorch Lightning is nothing more than organized PyTorch code.
+New to PyTorch Lightning and want to give it a go? Quick prototyping is our specility- you came to the right place!
 
-Once you've organized it into a LightningModule, it automates most of the training for you.
+Organizing your code with PyTorch Lightning makes your code:
+
+* more readable by decoupling the research code from the engineering
+* Easier to reproduce
+* less error prone by automtaing most of the training loop and tricky engineering
+* keep all the flexibility (this is all pure PyTorch), but removes a ton of boilerplate
+* scalable to any hardware without changing a thing!
 
 To illustrate, here's the typical PyTorch project structure organized in a LightningModule.
 
 .. figure:: https://pl-bolts-doc-images.s3.us-east-2.amazonaws.com/pt_animation_gif.gif
    :alt: Convert from PyTorch to Lightning
+   
+   
+In this guide we will go over how to build your Lightning code in 5 steps!
+
+
+----------
+
+Step 1: Install PyTorch Lightning
+---------------------------------
+
+We reccoment using conda environments
```

Comment:
Why do we recommend this? if you just use pip there, you can also use virtualenv. I think we should phrase it a bit different to not scare away the virtualenv users...

Maybe something like "While `virtualenv` also works fine, we will show some example setup with `conda` only". For newcomers, this gives them a starter, for experienced users this is fine, since they know, it's not only working with Conda but with python in general